spec/javascripts/support/jasmine_helper.rb

```diff
@@ -11,4 +11,5 @@
 # Prevent PhantomJS auto install, uses PhantomJS already on your path.
 Jasmine.configure do |config|
   config.prevent_phantom_js_auto_install = true
+  config.show_console_log = false
```

Comment:
Any reason for this change?

Reply:
Yeah, we added the config with a `true` to help debugging some stuff, and then disabled it (rather than removing it).